cargo-generate.toml

```diff
@@ -1,12 +1,12 @@
 [template]
 cargo_generate_version = "0.22"
+include = ["*.template"]
```

Comment:
Why did you end up calling these template instead of liquid? Shouldn't the latter have better syntax highlighting?

Reply:
If you call it `xyz.liquid`, `cargo-generate` will recognize it as a template for `xyz` and see that `xyz` already exists, so it will delete `xyz.liquid` instead. Well I don't know what `cargo-generate` is actually doing but that's the behavior I see.